tests/template_tests/syntax_tests/test_querystring.py

```diff
@@ -65,6 +70,14 @@ def test_querystring_remove(self):
         context = RequestContext(request)
         self.assertRenderEqual("querystring_remove", context, expected="?a=1")
 
+    @setup({"querystring_remove_dict": "{% querystring my_dict a=1 %}"})
+    def test_querystring_remove_from_dict(self):
+        request = self.request_factory.get("/", {"test": "value"})
+        template = self.engine.get_template("querystring_remove_dict")
+        context = RequestContext(request, {"my_dict": {"test": None}})
+        output = template.render(context)
+        self.assertEqual(output, "?a=1")
```

Comment:
I noticed that the newly added tests would not use the helper I landed in PR #18487. Could you please adapt each new tests (the ones for non errors) to use `assertRenderEqual`?